src/components/PhotoAnalysis/PhotoAnalysis.scss

```diff
@@ -0,0 +1,27 @@
+p.instructions{
+  font-style: italic;
+  font-size: large;
+}
+
+.cameraButton{
+  position: relative;
+  top: 425px;
```

Comment:
This will need to be changed. We need to find a way to position the `cameraButton` at the bottom regardless of the size of the content in the card (i.e. not with a fixed `px` value)

Reply:
Wouldn't a media query help this when we get to the responsive design section?